pkgs/snowflake-odbc/default.nix

```diff
@@ -0,0 +1,37 @@
+{ fetchurl, lib, stdenv, unixODBC }:
+
+let
+  version = "2.22.3";
+  sources = {
+    x86_64-linux = fetchurl {
+      url = "https://sfc-repo.snowflakecomputing.com/odbc/linux/latest/snowflake_linux_x8664_odbc-${version}.tgz";
+      sha256 = "65fceb95a4e44e25522801097b390bb4ec985248e8ceda55db08ee5620aac409";
+    };
+  };
+  unsupported = throw "Platform ${stdenv.hostPlatform.system} unsupported for aws-logs";
```

Comment:
Think this should be `unsupported for snowflake-odbc`, yes?

Reply:
Indeed it should.